understanding/20/three-flashes-or-below-threshold.html

```diff
@@ -11,12 +11,12 @@ <h1>Understanding Three Flashes or Below Threshold</h1>
    <section id="brief">
 		<h2>In brief</h2>
 		<dl>
-		<dt>Objective</dt><dd>Content does not trigger seizures</dd>
-		<dt>Author task</dt><dd>Avoid content that flashes, or keep it under thresholds</dd>
-		
-		</dl>
-		
-		</section>
+         <dt>Goal</dt><dd>Content does not trigger seizures.</dd>
+         <dt>What to do</dt><dd>Avoid content that flashes, or keep it under thresholds.</dd>
+         <dt>Why it's important</dt><dd>Flashing content can cause seizures.</dd> 
```

Comment:
@rainbreaw commented
> In understanding/20/three-flashes-or-below-threshold.html and understanding/20/three-flashes.html consider expanding: "migraines, dizziness, nausea, or even seizures."
```suggestion
         <dt>Why it's important</dt><dd>Flashing content can cause migraines, dizziness, nausea, or even seizures.</dd> 
```

Reply:
I like the mentioning. Yet I have no strong opinion whether this should be part of the in-brief section or (exclusively) part of the understanding.

Reply:
@GreggVan responded
> Slight tweak
I agree but I would delete   "even"    since that makes it sound like seizures are an afterthought or side benefit - when they are the primary driver.   Also it can do any of these so it should be "and".

Reply:
```suggestion
         <dt>Why it's important</dt><dd>Flashing content can cause migraines, dizziness, nausea, and seizures.</dd> 
```